CHANGELOG.md

```diff
@@ -739,7 +739,7 @@ A **huge** thanks to the **36 contributors** who submitted **148 PR's** since 0.
 * **Build** [1121](https://github.com/grafana/loki/pull/1121) **pracucci**: Publish loki-canary binaries on release
 * **Build** [1054](https://github.com/grafana/loki/pull/1054) **pstibrany**: Fix dep check warnings by running dep ensure
 * **Build/release** [1018](https://github.com/grafana/loki/pull/1018) **slim-bean**: updating the image version for loki-canary and adding the version increment to the release_prepare script
-* **Build/CI** [997](https://github.com/grafana/loki/pull/997) **slim-bean**: full cirlce
+* **Build/CI** [997](https://github.com/grafana/loki/pull/997) **slim-bean**: full circle
```

Comment:
lol this misspelling was intentional FWIW :wink:

Reply:
I'm pretty sure I looked at the PR and decided it didn't need to be misspelled.

If something is very clearly intentionally misspelled and doesn't make sense otherwise, I do leave it alone, e.g. a changelog entry of `spelling: cirlce -> circle`.

My personal preference is to not include misspellings in changelogs, unless it's announcing an API change.